### PR TITLE
Migrate schedule.json to TOML

### LINKED ISSUE #253: Improve formatting of schedule.json
Currently, the start and end times for periods in `public/schedule.json` are stored in milliseconds since midnight. However, we do not need to store information down to seconds, much less milliseconds. It would be more maintainable to store times as number of minutes since midnight (`hour * 60 + minute`). Or, even better, we could switch to [TOML](https://toml.io/en/), which is easier to read and write than JSON and offers native support for dates and times.

### PR DESCRIPTION
**Pull request by psvenk**
Closes #253. This helps readability and maintainability considerably. Some changes have been made to the frontend to work with native JavaScript `Date` objects instead of numbers.

### CODE REVIEW

**Comment by notrodes** [comment]
Can you add an error throw as a default case in the switch statement on line 178 of clock.js?

**Review by tektaxi — APPROVED** [review]
Works fine for me, but I haven't looked through the code much.

**Comment by psvenk** [comment]
@notrodes Are you referring to the `update_lunch` function? That is currently dormant code which will be addressed in #95 / #288 and is out of scope here. Are there any issues with the code that I wrote/touched in this pull request, or is this ready to merge?

File: public/js/clock.js [hunk 1]
```diff
@@ -29,6 +29,18 @@ let date_override = undefined;
 
 function schedulesCallback(response) {
     schedules = response;
+    // Convert all start and end times to JavaScript Date objects
+    for (const [, schedule] of Object.entries(schedules)) {
+        for (entry of schedule) {
+            // Convert each entry to a JavaScript Date object
+            let [, hours, minutes, seconds, ms]
```
Comment (notrodes) [inline public/js/clock.js]: Can you avoid this regex repeat?

Comment (psvenk) [reply public/js/clock.js]: Fixed in 2571fc5157.

**Review by notrodes — APPROVED** [review]

**Pull request merged**